### PR TITLE
Add riemann-portcheck

### PR DESCRIPTION
**Pull request by sdx23**
Enables for checking open tcp ports.

I'd be happy for all feedback/suggestions, haven't done much ruby yet.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -38,7 +38,8 @@ ship with the `riemann-tools` gem, including:
 * riemann-consul - Monitor Consul.
 * riemann-fd - Linux file descriptor use.
 * riemann-kvminstance - Monitor KVM instances.
-* riemann-ntp - Monitor NTP
+* riemann-ntp - Monitor NTP.
+* riemann-portcheck - Monitor open tcp ports.
```
Comment (jamtur01) [inline README.markdown]: TCP

File: bin/riemann-portcheck [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+require File.expand_path('../../lib/riemann/tools', __FILE__)
+
+class Riemann::Tools::Portcheck
+  include Riemann::Tools
+  require 'socket'
+
+  opt :hostname, "Host, defaults to localhost", :default => `hostname`.chomp
+  opt :ports, "List of ports to check, e.g. '-r 80 443'", :type => :ints
+
+  def initialize
```
Comment (jamtur01) [inline bin/riemann-portcheck]: What indentation are you using? I'd stick with 2 space per logic level.

File: bin/riemann-portcheck [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+          @hostname = opts.fetch(:hostname)
+          @ports = opts.fetch(:ports)
+  end
+
+  def tick
+    for thisport in @ports
+          portopen = Socket.tcp(@hostname, thisport, connect_timeout: 5) { true
```
Comment (jamtur01) [inline bin/riemann-portcheck]: This logic seems a little obtuse. Maybe a comment?

**Comment by sdx23** [comment]
Addressed your requests. Anything else?

**Comment by jamtur01** [comment]
Thanks!

**Pull request merged**